app/views/admin/volunteers/show.html.slim

```diff
@@ -121,6 +121,11 @@
             | Skills
           p = volunteer.skills
 
+          strong
+            = fa_icon 'wrench', class: 'margin-r-5'
```

Comment:
can we find a more suitable icon to represent the centers? like a group of buildings maybe?